### PR TITLE
RPS Challenge

### PR DESCRIPTION
**Pull request by alexleesonmill**
Relatively happy with the code.

I don't think it's fully tested, despite showing 100% coverage. Seems to be showing coverage of the tests rather than the actual code, and i'm too tired to work out why!

### CODE REVIEW

File: spec/unit/game_spec.rb [hunk 1]
```diff
@@ -14,4 +14,21 @@
     allow(game).to receive(:sample_move) { 'Scissors' }
     expect(game.single_match('Scissors')).to eq 'Draw!'
   end
+
```
Comment (margmartelo) [inline spec/unit/game_spec.rb:17]: Great that you used doubles (and stubbing)!

File: spec/unit/game_spec.rb [hunk 2]
```diff
@@ -14,4 +14,21 @@
...
+  it 'identifies a single player win' do
+    game.single_match('Scissors')
+    allow(game).to receive(:sample_move) { 'Paper' }
+    expect(game.single_match('Scissors')).to eq 'You won!'
+  end
+
+  it 'identifies a multi-player draw' do
```
Comment (margmartelo) [inline spec/unit/game_spec.rb:24]: Also great that you covered the multi-player possibility.

File: lib/game.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 class Game
   attr_reader :player_1, :player_2, :computer_move
 
-  MOVES = { 'Scissors' => 'Paper', 'Paper' => 'Rock', 'Rock' => 'Scissors' }
+  MOVES = { 'Scissors' => 'Paper', 'Paper' => 'Rock', 'Rock' => 'Scissors' }.freeze
```
Comment (margmartelo) [inline lib/game.rb:7]: Love how you present the moves as a hash!

File: views/index.erb [hunk 4]
```diff
@@ -1,25 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
   <title>Rock Paper Scissors!</title>
 </head>
+
```
Comment (margmartelo) [inline views/index.erb:10]: CSS, "refined" HTML... Wow! :D I really would like to see how it looks!

File: views/index.erb [hunk 5]
```diff
@@ -1,25 +1,28 @@
...
 <body>
   <h1>Rock Paper Scissors!</h1>
   <img src="http://mmafury.com/wp-content/uploads/2018/08/mma-rock-paper-scissors-gif.gif">
```
Comment (margmartelo) [inline views/index.erb:13]: Also like the implementation of images; it makes the game much more appealing!

File: views/play.erb [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
 
-<h2>Make your pick!</h2>
-<form action="/fight" method="post">
-  <input type="radio" name="player_1_move" value="Rock"/> Rock
-  <input type="radio" name="player_1_move" value="Paper"/> Paper
-  <input type="radio" name="player_1_move" value="Scissors"/> Scissors<br><br>
-  <button name="submit" value="Submit">Submit</button>
+  <h2>Make your pick!</h2>
+  <form action="/fight" method="post">
+    <input type="radio" name="player_1_move" value="Rock" /> Rock
```
Comment (margmartelo) [inline views/play.erb:16]: Type "radio"?! I'm guessing it's not relevant what we call it, right? :D

**Review by margmartelo — COMMENTED** [review]
Really organised! Tt's pleasant to read! 
Pertinent tests and I can see you cared to avoid repetition. Clever solution to the actual problem (rsp code). 
I'm really curious to see how it looks like! :)

File: lib/game.rb [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+  def sample_move
+    %w[Rock Paper Scissors].sample
+  end
+
+  def single_match(move)
+    @computer_move = sample_move
+    return 'Draw!' if move == computer_move
+
+    MOVES[move] == computer_move ? 'You won!' : 'You lost!'
```
Comment (alicelieutier) [inline lib/game.rb:22]: You are writing the logic to "who has won + draw" twice (once here, once ion line 30). You may want to extract that, as its a piece of logic that may evolve independently from the rest of the code in Lizard/Spock

**Review by alicelieutier — COMMENTED** [review]
Overall your code is well tested, and quite clean.
I would change a few small things, but this is a good attempt!

**Pull request closed without merging**